docs/content/how-to/node-tuning.md

```diff
@@ -0,0 +1,97 @@
+
+1. Create a ConfigMap which contains a valid Tuned manifest and reference it in a NodePool. The example Tuned manifest below defines a profile which sets `vm.dirty_ratio` to 55, on Nodes which contain the Node label  `tuned-1-node-label` with any value. 
+
+    Save the ConfigMap manifest in a file called `tuned-1.yaml`:
+    ```
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: tuned-1
+      namespace: clusters
+    data:
+      tuned: |
+        apiVersion: tuned.openshift.io/v1
+        kind: Tuned
+        metadata:
+          name: tuned-1
+          namespace: openshift-cluster-node-tuning-operator
+        spec:
+          profile:
+          - data: |
+              [main]
+              summary=Custom OpenShift profile
+              include=openshift-node
+
+              [sysctl]
+              vm.dirty_ratio="55"
+            name: tuned-1-profile
+          recommend:
+          - priority: 20
+            profile: tuned-1-profile
+    ```
+    > **_NOTE:_**  In the case where no labels are added to an entry in the `spec.recommend` section of the Tuned spec, NodePool based matching is assumed, so the highest priority profile in the `spec.recommend` section will be applied to Nodes in the pool. While more fine-grained Node label based matching is still possible by setting a label value in the Tuned `.spec.recommend.match`, users should be aware that Node labels will not persist during an upgrade, unless the NodePool `.spec.management.upgradeType` is set to `InPlace`.
+
+    Create the ConfigMap in the management cluster:
+    ```
+    oc --kubeconfig="$MGMT_KUBECONFIG" create -f tuned-1.yaml
+    ```
+
+    Reference the ConfigMap in the NodePools `spec.tunedConfig` field, either by editing an existing NodePool or creating a new NodePool. In this example we assume we only have one NodePool called `nodepool-1`, containing 2 Nodes.
+    ```
+    apiVersion: hypershift.openshift.io/v1alpha1
+    kind: NodePool
+    metadata:
+    ...
+    name: nodepool-1
+    namespace: clusters
+    ...
+    spec:
+    ...
+    tunedConfig:
+    - name: tuned-1
+    status:
+    ...
+    ```
+
+2. Now that the ConfigMap containing a Tuned manifest has been created and referenced in a NodePool, the Node Tuning Operator will sync the Tuned objects into the hosted cluster. You can check which Tuneds are defined, which profiles are set for each Node, and by labeling a Node with the label `tuned-1-node-label`, apply the custom profile we created to a Node.
```

Comment:
Please remove the relic `tuned-1-node-label`.  We no longer use it in this doc.

Reply:
Done.